ush/templates/FV3LAM_wflow.xml

```diff
@@ -84,18 +89,24 @@ tasks; and the "FCST" type is used for the RUN_FCST_TN task.
 <!ENTITY RSRV_HPSS    "<account>&ACCOUNT;</account><queue>&QUEUE_HPSS;</queue>">
 {%- endif %}
 {%- if partition_fcst is not none %}
-<!ENTITY RSRV_FCST    "<account>&ACCOUNT;</account><queue>&QUEUE_FCST;</queue><partition>{{ partition_fcst }}</partition>">
+<!ENTITY RSRV_FCST    "<account>&ACCOUNT;</account><queue>&QUEUE_FCST;</queue><partition>{{ partition_fcst }}</partition><native>&RRFS_RESERVE;</native>">
 {%- else %}
 <!ENTITY RSRV_FCST    "<account>&ACCOUNT;</account><queue>&QUEUE_FCST;</queue>">
 {%- endif %}
 <!ENTITY RSRV_GRAPHICS   "<account>&ACCOUNT;</account><queue>&QUEUE_GRAPHICS;</queue><partition>{{ partition_graphics }}</partition>">
+<!ENTITY RSRV_NCL   "<account>&ACCOUNT;</account><queue>&QUEUE_GRAPHICS;</queue><partition>{{ partition_graphics }}</partition>">
```

Comment:
Why not use RSRV_GRAPHICS here?

Reply:
We usually put the tasks before UPP into reservation and leave the graphic part out of reservation. The graphic doesn't impact the cycling and can be run in delayed mode. We put UPP into reservation mainly for downstream systems.

We can add graphic into reservation later if we have enough resources.

Reply:
I'm asking why we need to split RSRV_GRAPHICS (line 96) and RSRV_NCL (line 97).

Reply:
I deleted RSRV_NCL and ise RSRV_GRAPHICS and run them outside reservation.